README.md

```diff
@@ -128,6 +128,35 @@ click "Show all checks" in the info box at the bottom of the pull request page
 on GitHub, then choose "Details" for the "Test notebooks" job. From the job
 page, click "Summary", then download "Executed notebooks".
 
+### Ignoring warnings
+
+We don't want users to see warnings that can be avoided, so it's best to fix
+the code to avoid them. However, if a warning is unavoidable, you can stop it
+blocking CI by adding an `ignore-warning` tag to the cell. In VSCode,
+right-click the cell, choose "Add cell tag", type `ignore-warning`, then press
+"Enter". In Jupyter notebook (depending on version), choose View > Right
+Sidebar > Show Notebook Tools, then under "Common Tools" add a tag with text
+`ignore-warning`.
+
+### Extra code checks
+
+Our CI checks notebooks run from start to finish without errors or warnings.
+You can add extra checks in notebooks to catch other unexpected behavior.
+
+For example, say we claim a cell always returns the string `0011`. It would be
+embarassing if this was not true. We can assert this in CI by adding the
+following code cell, and hide it from users with a `remove-cell` tag.
+
+```python
+#| content: The output is `0011`.
```

Comment:
The comment is a bit confusing to me. What does `| content:` mean? Are you saying something like `# Double check the output is 0011`? If so, the code is self-documenting already and I'm not sure I agree with your suggestion to always add a comment.

Reply:
I imagined it would help tie code tests to actual claims in the prose, but I agree it's not super important. It might be good to have some kind of conventional comment in those cells to remind writers they're hidden from users (as tags aren't that visible).

Reply:
Agreed. How about:

```suggestion
# Confirm output is what we expect.
```

Reply:
Sure, that works. Thanks!